source/jormungandr/jormungandr/parking_space_availability/car/tests/star_test.py

```diff
@@ -39,6 +39,9 @@
     'properties': {'operator': 'Keolis Rennes', 'ref': '42'},
     'poi_type': {'name': 'parking relais', 'id': 'poi_type:amenity:parking'},
 }
+invalid_poi = {}
+
+empty_parking = ParkingPlaces(available=None, occupied=None, available_PRM=None, occupied_PRM=None)
```

Comment:
minor : `invalid_poi` and `empty_parking` seem to be used only once so they can be initialized in the function they are used in.